library/core/src/array/iter.rs

```diff
@@ -116,7 +115,7 @@ impl<T, const N: usize> IntoIter<T, N> {
     /// fn next_chunk<T: Copy, const N: usize>(
     ///     it: &mut impl Iterator<Item = T>,
     /// ) -> Result<[T; N], IntoIter<T, N>> {
-    ///     let mut buffer = MaybeUninit::uninit_array();
+    ///     let mut buffer = [MaybeUninit::uninit(); N];
```

Comment:
Ha, I was wondering why this was `Copy`.  Then I read more context and remembered I wrote this example in the first place 🙃

Reply:
Lol